### PR TITLE
Doc/ Update README with new job creation API

### PR DESCRIPTION
**Pull request by MatthieuMoreau0**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,12 +32,36 @@ pip install -e .[dev]
 ## Basic usage
 
 The package main component is a python object called `SDK` which can be used to create a `Batch` and send it automatically
-to Pasqal APIs using an API token generated in the [user portal](https://pasqal.cloud.io) (available soon).
+to Pasqal APIs using an API token generated in the [user portal](https://portal.pasqal.cloud).
 
-A `Batch` is a group of jobs with the same sequence that run on the same QPU.
+A `Batch` is a group of jobs with the same sequence that will run on the same QPU. For each job of a given batch you must set a value for each variables, if any, defined in your sequence.
```
Comment (anneclairelh) [inline README.md:37]: nit each *variable

File: README.md [hunk 2]
```diff
@@ -47,24 +71,18 @@ client_secret="your_client_secret" #Replace this value by the client secret of y
...
 sdk = SDK(client_id=client_id, client_secret=client_secret)
 
-# Creation of a sequence
-reg = Register.square(2, prefix="q")
-sequence = Sequence(reg, devices.Chadoq2)
-# You should add channels and variables to your sequence
-# See Pulser documentation for details
-serialized_sequence = sequence.serialize() # Serialize the sequence in json format
+# Set the wanted values for the variables defined in the sequence
+job1 = {"runs": 20, "variables": {"omega_max": 6} }
```
Comment (anneclairelh) [inline README.md:75]: Maybe a quick note specifying what `runs` is ?

File: README.md [hunk 3]
```diff
@@ -32,12 +32,36 @@ pip install -e .[dev]
...
+omega_max = sequence.declare_variable("omega_max")
+# Add a pulse to that channel with the amplitude omega_max
+generic_pulse = Pulse.ConstantPulse(100, omega_max, 2, 0.0)
+sequence.add(generic_pulse, "rydberg")
+
+# When you are done building your sequence, serialize it into a string
+serialized_sequence = sequence.serialize()
+```
+
+Once you have serialized your sequence, you can send it with the SDK with the following
```
Comment (anneclairelh) [inline README.md:63]: Maybe add "code: " at the end of this line?

File: README.md [hunk 4]
```diff
@@ -47,24 +71,18 @@ client_secret="your_client_secret" #Replace this value by the client secret of y
...
+# Set the wanted values for the variables defined in the sequence
+job1 = {"runs": 20, "variables": {"omega_max": 6} }
+job2 = {"runs": 50, "variables": {"omega_max": 10.5} }
 
-# Create a new batch and send it to Pasqal QPUs
-batch = sdk.create_batch(serialized_sequence)
-# You can also choose to run your batch on an emulator using the optional argument 'emulator'
-# batch = sdk.create_batch(serialized_sequence, emulator=True)
+# Send the batch of jobs to the QPU and wait for the results
+batch = s.create_batch(serialized_sequence, [job1,job2], wait=True)
```
Comment (anneclairelh) [inline README.md:79]: nit `sdk` ? We should probably test that exact script by the way  :) Should we add it to the end-to-end tests ?

Comment (anneclairelh) [reply README.md:79]: (not nit sorry :D )

File: README.md (hunk 2 above)
Comment (MatthieuMoreau0) [reply README.md:75]: done

File: README.md (hunk 4 above)
Comment (MatthieuMoreau0) [reply README.md:79]: Yes done.. I tested it initially and forgot to fix that issue my bad.. I'll rerun a test

Comment (MatthieuMoreau0) [reply README.md]: We already have a test very similar to this in end to end tests so not sure it brings much value to add it explicitly there. 

What we could do maybe is to create an `examples` folder (which could be used as tutorial for our users) in this repo that we would run in the e2e-tests. Thoughts ?

File: README.md [hunk 5]
```diff
@@ -71,12 +71,13 @@ client_secret="your_client_secret" #Replace this value by the client secret of y
 
 sdk = SDK(client_id=client_id, client_secret=client_secret)
 
-# Set the wanted values for the variables defined in the sequence
+# When creating a job, select a number of runs and set the wanted values for the variables
```
Comment (anneclairelh) [inline README.md:74]: nit "desired" instead of "wanted"

**Review by anneclairelh — APPROVED** [review]
LGTM!

**Review by anneclairelh — APPROVED** [review]

**Pull request merged**